# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-uvc/no-depmod.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-uvc-0.1.0-e/Makefile.vanilla	2006-10-28 04:12:49.000000000 +0200
+++ linux-uvc-0.1.0-e/Makefile	2007-03-30 11:46:12.934081631 +0200
@@ -18,7 +18,6 @@
 install:
 	@echo "Installing USB Video Class driver..."
 	@(cd $(KERNEL_DIR) && make -C $(KERNEL_DIR) SUBDIRS=$(PWD) INSTALL_MOD_DIR=$(INSTALL_MOD_DIR) modules_install)
-	depmod -ae
 
 clean:
 	-rm -f *.o *.ko .*.cmd .*.flags *.mod.c Modules.symvers
